hazelcast@5.X.rb

```diff
@@ -0,0 +1,29 @@
+class HazelcastAT5X < Formula
+    desc "Tool to run Hazelcast IMDG member instances locally"
+    homepage "https://github.com/hazelcast/hazelcast-command-line"
+    url "https://github.com/hazelcast/hazelcast-command-line/releases/download/v5.2021.07.1/hazelcast-5.0-BETA-1.tar.gz"
+    sha256 "f108d22a1aec61bbd637f89ff522af7d9861ef13afcfad24a5095127f04f091d"
+  
+    bottle :unneeded
+  
+    depends_on "openjdk" => :recommended
+  
+    def install
+      libexec.install Dir["*"]
+      Dir["#{libexec}/bin/hz*"].each do |path|
+        executable_name = File.basename(path)
+        if executable_name.end_with? ".bat"
+          next
+        end
+        (bin/executable_name).write_env_script libexec/"bin/#{executable_name}", Language::Java.overridable_java_home_env
```

Comment:
I have zero Homebrew knowledge, so I have no idea how this works exactly, but - do we need the same as for deb/rpm packages - to remove the links during uninstall?

Reply:
No we don't need to explicitly remove symlinks, they are removed automatically by the Homebrew